muselsl/muse.py

```diff
@@ -124,6 +106,9 @@ def ask_control(self):
         "ps": preset selected
         "rc": return status, if 0 is OK
         """
+        if self.backend == 'bluemuse':
+            print('Not supported by bluemuse backend.')
+            return
```

Comment:
i would use a warning in case someone was logging

Reply:
Jason added a helper